pages/cos.py

```diff
@@ -6,7 +6,6 @@
 
 
 class COSDonatePage(BasePage):
-    url = 'https://cos.io/about/support-cos/'
+    url = 'https://www.cos.io/support'
 
-    identity = Locator(By.CSS_SELECTOR, 'div[class="slide-wrapper-background"]', settings.LONG_TIMEOUT)
-    page_heading = Locator(By.CSS_SELECTOR, 'div[class="banner-element"]')
+    identity = Locator(By.XPATH, '//meta[@name="cos:id" and @content="donate-page"]', settings.LONG_TIMEOUT)
```

Comment:
Why are we leaving `identity` in place now that this check has been moved to a utility method?  Shouldn't this be removed along w/ `page_heading`?

Reply:
I asked Josh to keep it since the identity isn't wrong here. The meta tag is indeed the identity of the page. We need to improve the [locator](https://github.com/cos-qa/osf-selenium-tests/blob/b7f3f21376b7d6f751993cdcffea9262856263e3/base/locators.py#L151) so we no longer need the [path hack](https://github.com/cos-qa/osf-selenium-tests/pull/107/files#diff-2807b7734eaaec258019c01f87e6c920R262). However, it is out of the scope of this ticket.